source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -268,21 +283,98 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
+                                                        winrt::TileControl tileControl)
+    {
+        auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
+        auto weakImageSource{winrt::make_weak(svgImageSource)};
+        auto weakTileControl{winrt::make_weak(tileControl)};
+
+        if (imageUrl.SchemeName() == L"data")
+        {
+            auto imagePath = HStringToUTF8(imageUrl.Path());
+            auto foundBase64 = imagePath.find("base64");
+            winrt::DataWriter dataWriter{winrt::InMemoryRandomAccessStream{}};
+
+            if (foundBase64 != std::string::npos)
+            {
+                // Decode base 64 string
+                std::string data = AdaptiveBase64Util::ExtractDataFromUri(imagePath);
+                std::vector<char> decodedData = AdaptiveBase64Util::Decode(data);
+                dataWriter.WriteBytes(std::vector<byte>{decodedData.begin(), decodedData.end()});
+            }
+            else
+            {
+                std::string data = ExtractSvgDataFromUri(imageUrl);
+                dataWriter.WriteBytes(std::vector<byte>{data.begin(), data.end()});
+            }
+                
+            co_await dataWriter.StoreAsync();
+            if (auto strongImageSource = weakImageSource.get())
+            {
+                auto stream = dataWriter.DetachStream().try_as<winrt::InMemoryRandomAccessStream>();
+                stream.Seek(0);
+                auto sizeParseOp{ParseSizeOfSVGImageFromStreamAsync(stream)};          
+                co_await wil::resume_foreground(GetDispatcher(strongImageSource));
+                auto size = co_await sizeParseOp;
+                strongImageSource.RasterizePixelHeight(size.Height);
+                strongImageSource.RasterizePixelWidth(size.Width);
+                co_await strongImageSource.SetSourceAsync(stream); 
+                winrt::Image image;
+                image.Source(strongImageSource);
+                if (auto strongTileControl = weakTileControl.get())
+                {
+                    strongTileControl.LoadImageBrush(image);
+                }
+            }
+            co_return;
+        }
+
+        winrt::HttpClient httpClient;
+        auto getOperation = co_await httpClient.GetAsync(uri);
+        auto readOperation = co_await getOperation.Content().ReadAsStringAsync();
+        auto size{ParseSizeOfSVGImageFromXmlString(readOperation)};
+
+        if (auto strongImageSource = weakImageSource.get())
+        {
+            co_await wil::resume_foreground(GetDispatcher(strongImageSource));
+            strongImageSource.RasterizePixelHeight(size.Height);
+            strongImageSource.RasterizePixelWidth(size.Width);
+            winrt::Image image; 
+            image.Source(strongImageSource);
+            strongImageSource.UriSource(imageUrl);
+            if (auto strongTileControl = weakTileControl.get())
+            {
+                strongTileControl.LoadImageBrush(image);
+			}
```

Comment:
nit:indentation

Reply:
Looks fine in visual studio.

![image](https://github.com/microsoft/AdaptiveCards/assets/165726045/3e750fc4-d72c-41ef-af8d-acd66de38bb6)